weather_mv/loader_pipeline/sinks.py

```diff
@@ -514,3 +516,24 @@ def open_dataset(uri: str,
         beam.metrics.Metrics.counter('Failure', 'ReadNetcdfData').inc()
         logger.error(f'Unable to open file {uri!r}: {e}')
         raise
+
+
+def get_file_time(element: t.Any) -> int:
+    try:
+        if element.startswith("gs://"):
+            bucket_name, file_name = element[5:].split("/", 1)
+
+            client = storage.Client()
+            bucket = client.get_bucket(bucket_name)
+            blob = bucket.get_blob(file_name)
+
+            updated_time = int(blob.updated.timestamp())
+        else:
```

Comment:
Why if else ?

Reply:
So when uri is a local file, it still works.

Reply:
Got it! 

Let's add a comment for posterity.